src/Packages/Passport/Samples~/PassportManagerPrefab/README.md

```diff
@@ -0,0 +1,229 @@
+# PassportManager Prefab
+
+The **PassportManager** is a drag-and-drop prefab that provides an easy way to integrate Immutable Passport authentication into your Unity game.
+
+## 🚀 Quick Start
+
+### Option 1: Just Authentication (Code-based)
+
+1. **Import the Sample**: In Unity Package Manager, find "Immutable Passport" and import the "PassportManager Prefab" sample
+2. **Drag the Prefab**: Drag `PassportManager.prefab` from the Samples folder into your scene
+3. **Configure Settings**: In the Inspector, set your Client ID and redirect URIs
+4. **Use Events or Code**: Subscribe to events or call methods directly
+5. **Test**: Hit Play - the prefab will automatically initialize Passport!
+
+### Option 2: Complete UI Integration (No Code Required!)
+
+1. **Import the Sample**: Same as above
+2. **Drag the Prefab**: Same as above  
+3. **Configure Settings**: Same as above
+4. **Add PassportUIController**: Add the `PassportUIController` script to a GameObject and assign your UI elements (supports both Legacy Text and TextMeshPro)
+5. **Wire Events**: In PassportManager Inspector, connect the events to PassportUIController methods
+6. **⚠️ Production Note**: Review cursor management settings for your game type
+7. **Test**: Hit Play - full authentication flow with automatic UI management!
+
+## ⚙️ Configuration
+
+### Required Settings
+
+- **Client ID**: Your Immutable Passport client ID
+- **Redirect URI**: Authentication callback URL (e.g., `mygame://callback`)
+- **Logout Redirect URI**: Logout callback URL (e.g., `mygame://logout`)
+
+### Optional Settings
+
+- **Environment**: `SANDBOX` (default) or `PRODUCTION`
+- **Auto Initialize**: Automatically initialize on Start (default: true)
+- **Auto Login**: Automatically attempt login after initialization (default: false)
+- **Direct Login Method**: Pre-select login method (None, Google, Apple, Facebook)
```

Comment:
Rename this to default login method? So it's easier to understand without having to read the README